tests/unit/amo/sagas/test_users.js

```diff
@@ -222,8 +222,8 @@ describe(__filename, () => {
     it('can receive a non-empty notifications object (dict) in payload', async () => {
       const state = sagaTester.getState();
 
-      const username = 'babar';
-      const user = createUserAccountResponse({ id: 5001, username });
+      const userId = 5001;
+      const user = createUserAccountResponse({ id: userId, username: 'babar' });
```

Comment:
Why do you need `username: 'babar'` here?

Reply:
I don't think it changes anything.